src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -51,5 +64,6 @@ public double getPosition() {
   @Override
```

Comment:
we should add a method called ```public void setTargetPosition(double position)``` which stores the value to a class member variable.